packages/legacy/core/App/screens/Home.tsx

```diff
@@ -130,7 +131,15 @@ const Home: React.FC<HomeProps> = () => {
         scrollEnabled={notifications?.length > 0 ? true : false}
         decelerationRate="fast"
         ListEmptyComponent={NoNewUpdates}
-        ListHeaderComponent={() => <HomeHeaderView />}
+        ListHeaderComponent={() => {
+          const homeHeaderViewProps = {
+            notifications: notifications,
+            showNotifications: showNotifications,
+          }
+          return (
+            <HomeHeaderView {...homeHeaderViewProps} />
+          )
+        }}
```

Comment:
```suggestion
        ListHeaderComponent={() => <HomeHeaderView notifications={notifications} /> }
```

If the `showNotifications` gets moved into config, this might be a bit cleaner